graph/formats/dot/internal/parser/parser_test.go

```diff
@@ -86,6 +88,38 @@ func TestParseFile(t *testing.T) {
 	}
 }
 
+func TestParseFuzz(t *testing.T) {
+	r, err := zip.OpenReader("../../fuzz/corpus.zip")
+	if err != nil {
+		if os.IsNotExist(err) {
+			t.Skip("no corpus")
+		}
+		t.Fatalf("failed to open corpus: %v", err)
+	}
+	defer r.Close()
+
+	for _, f := range r.File {
+		rc, err := f.Open()
+		if err != nil {
+			t.Fatalf("failed to open %q: %v", f.Name, err)
+		}
+		func() {
```

Comment:
What is the purpose of this anonymous function? Is it just to capture the scope of the local variables? It does not seem to be used in either a Go-routine statement or a defer statement.

Reply:
This is to allow recovery of panics for each individual test case. Otherwise only the first panic would be found.

Reply:
Ok, thanks. I learnt something new :)